.github/workflows/ci.yaml

```diff
@@ -0,0 +1,117 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'main'
+
+env:
+    SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
+
+jobs:
+    composer-validate:
+        name: "Composer validate (${{ matrix.php-version }})"
+
+        runs-on: ubuntu-16.04
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.4'
+
+        steps:
+            -
+                name: "Checkout code"
+                uses: "actions/checkout@v2"
+
+            -
+                name: "Install PHP"
+                uses: "shivammathur/setup-php@v2"
+                with:
+                    coverage: "none"
+                    php-version: "${{ matrix.php-version }}"
+                    tools: "composer:v2"
+
+            -
+                name: "Validate composer.json"
+                run: "composer validate --strict --no-check-lock"
+
+            -
+                name: "Composer install"
+                uses: "ramsey/composer-install@v1"
+                with:
+                    composer-options: "--no-scripts"
+
+    tests:
+        needs:
+            - "composer-validate"
+
+        name: "PHP ${{ matrix.php-version }} + ${{ matrix.dependencies }}"
+
+        runs-on: ubuntu-16.04
+
+        continue-on-error: ${{ matrix.allow-failures }}
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version:
+                    - '7.1'
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                dependencies:
+                    - 'highest'
+                allow-failures: [false]
+                include:
+                    - php-version: '7.1'
+                      dependencies: 'lowest'
+                      allow-failures: false
+
+        steps:
+            - name: "Checkout code"
+              uses: actions/checkout@v2.3.3
+
+            - name: "Install PHP with extensions"
+              uses: shivammathur/setup-php@2.7.0
+              with:
+                  coverage: "none"
+                  php-version: ${{ matrix.php-version }}
+                  tools: composer:v2
+
+            - name: "Add PHPUnit matcher"
+              run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+            - if: matrix.php-version == '8.0'
+              run: composer config platform.php 7.4.99 || true
```

Comment:
do we still need that now that PHP 8 is stable or are the relevant dependencies properly compatible ?

Reply:
I don't know

Reply:
I suggest trying to remove it and see if it works.

Reply:
Done and pushed ⏳

Reply:
No it fails: https://github.com/OskarStark/flex/pull/1/checks?check_run_id=1463674419

I reverted the commit

Reply:
I opened https://github.com/composer/composer/issues/9523

Reply:
Thanks

Reply:
@stof I removed the step and it looks like its working 👍